src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
Usually when I add a unit test to verify a fix, I tag the test with the issue number. I thought the tag should last forever as a reminder of why the test is there. This is different than the comments identifying why some workaround is employed. In any case this is inconsistent with your leaving the bug ref for 10574 above